<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulk Club MS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
	color: #00B2FF;
	background-color: white;
}

#header_label {
	color: #0088FF;
}
#id_input {
	color: #00B2FF;
}

#id_label {
	color: #00B2FF;
}

#login_btn {
	background-color: #00B2FF;
	color: white;
}

#psw_input {
	color: #00B2FF;
}

#psw_label{
	color: #00B2FF;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="header_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>552</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Welcome to the Bulk Club Managment System</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="login_form">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Please log in</string>
    </property>
    <widget class="QPushButton" name="login_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QLabel" name="psw_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Give your password </string>
     </property>
    </widget>
    <widget class="QLabel" name="id_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Give your ID               </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="psw_input">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_input">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#id_input {
	border-color: blue;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="mainMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="report">
     <attribute name="title">
      <string>Report</string>
     </attribute>
     <widget class="QTextBrowser" name="display">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelWeekday">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose the weekday:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRevenue">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>170</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Revenue:</string>
      </property>
     </widget>
     <widget class="QLabel" name="revenue">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="showExec">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>220</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Executive Members</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="showRegular">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Regular Members</string>
      </property>
     </widget>
     <widget class="QLabel" name="numberExec">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelExec">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>290</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Executive Members:</string>
      </property>
     </widget>
     <widget class="QLabel" name="numberRegular">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>370</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRegular">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>350</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Regular Members:</string>
      </property>
     </widget>
     <widget class="QLabel" name="login_header">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>552</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="display_btn">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
     <widget class="QComboBox" name="weekdayBox">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Monday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thursday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Friday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saturday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sunday</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>This section lets you
generate the day report</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="all_purch">
     <attribute name="title">
      <string>All purchases</string>
     </attribute>
     <widget class="QTextBrowser" name="display_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="display_btn_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>This section displays all 
purchases sorted by 
membership no.</string>
      </property>
     </widget>
     <widget class="QLabel" name="grandTotal">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGrandTotal">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>190</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Grand total:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="items_sold">
     <attribute name="title">
      <string>Items sold</string>
     </attribute>
     <widget class="QTextBrowser" name="display_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelTotalRev">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Total revenue:</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalRevenue">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>330</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>This section displays each
item sold sorted by quantity</string>
      </property>
     </widget>
     <widget class="QPushButton" name="display_btn_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ItemsComboBox">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ALL ITEMS</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="searchByItemsName">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_searchByItemsName">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Searching by item's name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="members">
     <attribute name="title">
      <string>Members</string>
     </attribute>
     <widget class="QTextBrowser" name="display_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>This section displays  
members (using filters)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="optionBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>150</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Sort by rebate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sort by ID</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelRevenue_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose an option:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRevenue_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Membership expiring in:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="expiryBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="display_btn_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
     <widget class="QPushButton" name="filter_btn">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRevenue_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>320</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Add/Delete members</string>
      </property>
     </widget>
     <widget class="QComboBox" name="addDelete_combo">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>350</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Add</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delete</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="addDelete_btn">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add/Delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineName">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>400</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineExecutive">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>440</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Executive?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineExpiry">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>480</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Expiry</string>
      </property>
     </widget>
     <widget class="QComboBox" name="delete_combobox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>430</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <zorder>delete_combobox</zorder>
     <zorder>display_4</zorder>
     <zorder>label_4</zorder>
     <zorder>optionBox</zorder>
     <zorder>labelRevenue_4</zorder>
     <zorder>labelRevenue_5</zorder>
     <zorder>expiryBox</zorder>
     <zorder>display_btn_4</zorder>
     <zorder>filter_btn</zorder>
     <zorder>labelRevenue_9</zorder>
     <zorder>addDelete_combo</zorder>
     <zorder>addDelete_btn</zorder>
     <zorder>lineName</zorder>
     <zorder>lineExecutive</zorder>
     <zorder>lineExpiry</zorder>
    </widget>
    <widget class="QWidget" name="inventory">
     <attribute name="title">
      <string>Inventory</string>
     </attribute>
     <widget class="QTextBrowser" name="display_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="addDeleteInventory">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Add item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delete item</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>This section lets you
add and delete items</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRevenue_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Add or delete?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemName">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelRevenue_7">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name of the item:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRevenue_8">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>260</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Gross price for the item:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="grossPrice">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>290</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="display_btn_5">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Add/Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="personal_data">
     <item>
      <widget class="QLabel" name="admin_man">
       <property name="text">
        <string>Admin/Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="id">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>20</width>
      <height>421</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
